core/src/CMakeLists.txt

```diff
@@ -60,6 +60,9 @@ set(HEADERS_IMPL
 
 add_library(cabanacore INTERFACE)
 
+# Require minimum of C++17
+set_target_properties(cabanacore PROPERTIES INTERFACE_COMPILE_FEATURES cxx_std_17)
```

Comment:
As I understood from discussion Cajita will inherit this since it links to the core, but I'm not sure

Reply:
I would still make it explicit!

Reply:
Done